package.json

```
@@ -33,29 +33,29 @@
     "dist"
   ],
   "scripts": {
-    "build": "nuxt-module-build",
-    "dev": "nuxt-module-build --stub && nuxi dev .playground",
-    "dev:build": "nuxt-module-build --stub && nuxi build .playground",
+    "build": "nuxt-module-build build build",
```

```suggestion
    "build": "nuxt-module-build build",
```

I don't think we need two of them!